types/store.go

```diff
@@ -155,15 +153,27 @@ type (
 	GasConfig = types.GasConfig
 )
 
-func NewGasMeter(limit Gas) GasMeter {
-	return types.NewGasMeter(limit)
-}
-
 type (
 	ErrorOutOfGas    = types.ErrorOutOfGas
 	ErrorGasOverflow = types.ErrorGasOverflow
 )
 
+func NewGasMeter(limit Gas) GasMeter {
+	return types.NewGasMeter(limit)
+}
+
 func NewInfiniteGasMeter() GasMeter {
 	return types.NewInfiniteGasMeter()
 }
+
+func NewSnapshotOptions(interval uint64, keepRecent uint32) snapshottypes.SnapshotOptions {
```

Comment:
These three functions are just wrappers around `snapshottypes` and `pruningtypes`. Why not just define function aliases?

Reply:
I decided to remove these in general.

One of the reasons is that I observed in many test files that we would import both store types as `sdk` and, for example, pruning types as `pruningtypes`. Then, we would sometimes use `sdk.NewPruningOptions(...)`, and in other times `pruningtypes.NewPruningOptions(...).

I think these wrappers or aliases may add unnecessary complexity and redundancy. In the beginning, I was confused by their purpose but decided to follow the pattern from this repository in my PR. However, after iterating on this for a while, I think that it is cleaner to avoid having these wrappers or aliases without a clear purpose.